apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant/Defendant.tsx

```diff
@@ -253,11 +255,17 @@ const Defendant: React.FC = () => {
               )
             }}
             value={
-              workingCase.indictmentSubType
+              workingCase.indictmentSubTypes &&
+              Object.keys(workingCase.indictmentSubTypes).length > 0
                 ? {
-                    value: IndictmentSubType[workingCase.indictmentSubType],
+                    value:
+                      IndictmentSubType[
+                        Object.entries(workingCase.indictmentSubTypes)[0][1][0]
```

Comment:
This is unclear to me. Please consider writing a helper function for this and explain the [0][1][0] thing with a comment

Reply:
This is throwaway code, so I would rather not do that. The whole logic will change when the client implements multiple indictment sub-types.